index.src.html

```diff
@@ -2052,111 +2072,140 @@
 </pre>
 
 ### Internal Slots ###{#videoframe-internal-slots}
-<dl>
-  <dt><dfn attribute for=VideoFrame>\[[detached]]</dfn></dt>
-  <dd>
-    Boolean indicating whether {{destroy()}} was invoked and underlying
-        resources have been released.
-  </dd>
-</dl>
+: <dfn attribute for=VideoFrame>\[[detached]]</dfn></dt>
+:: A boolean indicating whether {{VideoFrame/close()}} was invoked.
+
+: <dfn attribute for=VideoFrame>[[resource reference]]</dfn>
+:: A reference to the [=frame resource=] that stores the pixel data for
+    this frame.
 
 ### Constructors ###{#videoframe-constructors}
 
-NOTE: this section needs work. Current wording assumes a VideoFrame can always
-    be easily represented using one of the known pixel formats. In practice, the
-    underlying UA resources may be GPU backed or formatted in such a way that
-    conversion to an allowed pixel format requires expensive copies and
-    translation. When this occurs, we should allow planes to be null and format
-    to be "opaque" to avoid early optimization. We should make conversion
-    explicit and user controlled by offering a `videoFrame.convertTo(format)`
-    that returns a Promise containing a new VideoFrame for which the
-    copies/translations are performed.
-
-<dfn constructor for=VideoFrame title="VideoFrame(imageBitmap, frameInit)">
-  VideoFrame(imageBitmap, frameInit)
+<dfn constructor for=VideoFrame title="VideoFrame(imageBitmap, init)">
+  VideoFrame(imageBitmap, init)
```

Comment:
We really need to do something about color spaces here. What's the best issue to track this?

Reply:
We're definitely missing proper handling of color. Discussed in https://github.com/w3c/webcodecs/issues/47